server/libs/infra/src/db/entities/system-config.entity.ts

```diff
@@ -40,7 +40,7 @@ export interface SystemConfig {
     targetVideoCodec: string;
     targetAudioCodec: string;
     targetScaling: string;
-    transcodeAll: boolean;
+    transcode: string;
```

Comment:
It would be better to use an enum type instead of a string for this setting

Reply:
Done!